ContactForm.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7" lang=""> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8" lang=""> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9" lang=""> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js" lang="">
+<!--<![endif]-->
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <style>
+        body {
+            padding-top: 50px;
+            padding-bottom: 20px;
+        }
+    </style>
+    <link rel="stylesheet" href="css/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="css/main.css">
+
+    <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+</head>
+
+<div class="bootstrap-iso">
+<div class="container-fluid">
+ <div class="row">
+  <div class="col-md-6 col-sm-6 col-xs-12">
+   <form method="post">
+    <div class="form-group ">
+        <h2>Contact Us</h2>
+     <label class="control-label " for="name">
+     <form action="mailto:someone@example.com" method="post" enctype="text/plain"></form>        
+      Name
+     </label>
+     <input class="form-control" id="name" name="name" type="text"/>
+    </div>
+    <div class="form-group ">
+     <label class="control-label requiredField" for="email">
+      Email
+      <span class="asteriskField">
+       *
+      </span>
+     </label>
+     <input class="form-control" id="email" name="email" type="text"/>
+    </div>
+    <div class="form-group ">
```

Comment:
Good job, everything looks great. Only thing I could catch was some extras spaces behind a few classes in some div and label tags.